pmpro-reports-dashboard.php

```diff
@@ -103,7 +103,7 @@ function pmprordb_redirect_canonical_callback( $redirect_url, $requested_url ) {
 function pmprordb_add_links_report_page() {
 
 	// Only load on the reports main page.
-	if ( ! isset( $_REQUEST['page'] ) &&  $_REQUEST['page'] != 'pmpro-reports' ) {
+	if ( ! isset( $_REQUEST['page'] ) ||  $_REQUEST['page'] != 'pmpro-reports' ) {

```

Comment:
changed this one here @andrewlimaza otherwise i had a warning due the key doesn't exist when page is not set